buildtest/cli/buildspec.py

```diff
@@ -846,19 +846,40 @@ def print_invalid_buildspecs(self, error=None):
         Args:
             error (bool, optional): Display error messages for invalid buildspecs. Default is ``False`` where we only print list of invalid buildspecs
         """
-
-        table = Table(
-            "Buildspec",
-            title="Invalid Buildspecs",
-            header_style="blue",
-            style="cyan",
-            title_style="red",
-            row_styles=["red"],
-        )
-
+        table = Table()
+        if(not self.terse and not self.header):
+            table = Table(
+                "Buildspec",
+                title="Invalid Buildspecs",
+                header_style="blue",
+                style="cyan",
+                title_style="red",
+                row_styles=["red"],
+            )
+        if(not self.terse and self.header):
+            table = Table(
+                    "Buildspec",
+                    style="cyan",
+                    title_style="red",
+                    row_styles=["red"],
+                )
+        if(self.terse and not self.header):
+            table = ["Invalid Buildspecs"]
+                
+        if(self.terse and self.header):
+            table = []
```

Comment:
@shahzebsiddiqui I think that i have resolved the issue. I am waiting till commit/push my code. Quick question; in your last hint are you saying that the error output should not be displayed if the terse option is given?. If that is the case what should be the output of this command `buildtest bc find --terse invalid --error`. I ask this because as the code was before my changes if the `--error` flag is used only the error codes are printed. But if the `--error` flag is used and so is `--terse` and they way that i have read your comment it seems that there should be no output. Is this right?